app/unit-tests/src/org/commcare/android/tests/activities/UpdateActivityTest.java

```diff
@@ -64,27 +65,33 @@ public void invalidUpdateTest() {
         ShadowActivity shadowActivity = Shadows.shadowOf(updateActivity);
         shadowActivity.clickMenuItem(UpdateActivity.MENU_UPDATE_FROM_CCZ);
 
-        // make sure there are no pinned notifications
+        // Make sure there are no pinned notifications before we start
         NotificationManager notificationManager =
-                (NotificationManager)RuntimeEnvironment.application.getSystemService(Context.NOTIFICATION_SERVICE);
-        Notification notification = Shadows.shadowOf(notificationManager).getNotification(CommCareNoficationManager.MESSAGE_NOTIFICATION);
+                (NotificationManager)RuntimeEnvironment.application
+                        .getSystemService(Context.NOTIFICATION_SERVICE);
+        notificationManager.cancel(CommCareNoficationManager.MESSAGE_NOTIFICATION);
```

Comment:
Nice catch, sounds very unfortunate - usually I'd expect the test runner to give you a clean sandbox for each run, though maybe the DVM spin-up makes that unrealistic for Roboelectric. If we run into this again maybe we should genericize what you've done to run between each test